cli-feed-v4.json

```diff
@@ -56,7 +56,7 @@
       "hidden": false
     },
     "v0-prerelease": {
-      "release": "4.84.0-inprocess",
+      "release": "4.86.0-inprocess",
```

Comment:
Do we want to skip `4.85-inprocess` here? Checking if that's the process we want to follow.

Reply:
Release tags are not related to core tools or host versions. They are just used internally. Making this change so the numbers match. Making it easier to review and spot if anything has gone out of sync